src/assets/database.sql

```diff
@@ -366,6 +366,44 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
 ('Zerocopy', '(client only) Use a "zero copy" method of sending data', 'iperf3 --zerocopy', 19, 'checkbox', '--zerocopy', NULL),
 ('Dont fragment', '(client only) Set the IPv4 Dont Fragment (DF) bit on outgoing packets', 'iperf3', 19, 'checkbox', '--dont-fragment', NULL);
 
+--
+-- Populate data for `commands`: dig
+--
+INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `command`, `example`, `sudo`, `category`) VALUES 
+('DNS Server', 'Defines the DNS server which will be used in the command', 'dig @8.8.8.8 example.hackinsdn.ubfa,br', 27, 'input', NULL, NULL, NULL, 'BASIC OPTIONS'),
+('DNS domain to query', 'Set the DNS domain which will be queried', 'domain example.hackinsdn.ufba.br', 27, 'input', 'domain', NULL, NULL, 'BASIC OPTIONS'),
```

Comment:
Domain to query should be the target parameter of this tool, not an optional one.

Reply:
Note in your screenshot `domain` being on command line on bulk execution page.